src/chains/ethereum/ethereum/src/api-types.ts

```diff
@@ -194,4 +194,22 @@ export namespace Ethereum {
 
   // whisper
   export type WhisperPostObject = UtilTypes.WhisperPostObject;
+
+  //#region eth_getProof
+  export type StorageProof = {
```

Comment:
I've aligned this with the Geth implementation, which differs from the specification in [EIP-1186](https://eips.ethereum.org/EIPS/eip-1186#specification).

The specification defines the key as `Quantity` type. When I call `eth_getProof` on infura, specifying storage keys, I receive the keys back as a fixed length, (the value _is_ returned as a `Quantity` type as expected [the returned value is not forced to an even number of octets]). 

ie.
```
{
  key: "0x0000000000000000000000000000000000000000000000000000000000000001",
  value: "0x1",
  proof: [
    "0xf8518080a0a75fd430c15087d0a2c54dcfe4bfa47c13129638e5ec6ea8236bd5df2ad66ac68080808080808080a0f4984a11f61a2921456141df88de6e1a710d28681b91af794c5a721e47839cd78080808080",
    "0xe2a0310e2d527612073b26eecdfd717e6a320cf44b4afac2b0732d9fcbe2b7fa0cf601"
  ]
}
```